module-1/src/Components/MovieForm/MovieForm.tsx

```diff
@@ -0,0 +1,120 @@
+import "./MovieForm.css";
+interface IMovieForm {
+  action: string;
+  movie?: Movie;
+}
+
+export const MovieForm = ({ action, movie }: IMovieForm) => {
+  return (
+    <div className="formSettings bg-custom_dark_gray  min-w-full">
+      <h1 className="ml-5 mb-2 text-white text-2xl font-Montserrat">
+        {action} MOVIE
+      </h1>
+      <div className="grid grid-cols-2 gap-3 mx-5 pb-3">
+        <div className=" grid col-auto">
+          <label className="text-custom_pink font-Montserrat " htmlFor="Title">
+            Title
+          </label>
+          <input
+            className="bg-custom_light_gray text-white p-1"
+            type="text"
+            name="Title"
+            id="Title"
+            value={action === "ADD" ? "" : movie?.movieName}
```

Comment:
`action` should not define input's value, it should be solely driven by `movie.<prop>`. For new movies the prop will be empty, for edit there will be an initial value.

Reply:
Fixed